docs/reference/src/language/concepts/translations.md

```diff
@@ -1,21 +1,38 @@
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
 # Translations
 
-Use Slint's translation infrastructure to make your application available in different languages.
+Slint's translation infrastructure makes your application available in different languages.
+
+## Prerequisites
+
+Install the `slint-tr-extractor` tool to extract translatable strings from `.slint` files:
+
+```sh
+cargo install slint-tr-extractor
+```
+
+## Overview
 
 Complete the following steps to translate your application:
 
-1. Identify all user visible strings that need to be translated and annotate them with the `@tr()` macro.
-2. Extract annotated strings using the `slint-tr-extractor` tool and generate `.pot` files.
+<!-- TODO: Link up -->
```

Comment:
Is this still a TODO?

Reply:
I can't quite remeber what I was thinking of here now. Maybe linking to the macro definitions